### PR TITLE
First draft for pVACtools documentation

### LINKED ISSUE #7: Create pVACtools docs

### PR DESCRIPTION
**Pull request by susannasiebert**
Closes #7

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -1,37 +1,37 @@
 Installation
 ============
 
-pVAC-Seq is written for Linux but some users have been able to run it successfully on Mac OS X. If you are using Windows you will need to set up a Linux environment, for example by setting up a virtual machine.
+pVACtools is written for Linux but some users have been able to run it successfully on Mac OS X. If you are using Windows you will need to set up a Linux environment, for example by setting up a virtual machine.
 
-pVAC-Seq requires Python 3.5. Before running any installation steps check the Python version installed on your system:
+pVACtools requires Python 3.5. Before running any installation steps check the Python version installed on your system:
```
Comment (chrisamiller) [inline docs/install.rst]: s/steps check/steps, check/

File: docs/pvacfuse.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    :align: right
+    :alt: pVACfuse logo
+
+pVACfuse
+====================================
+
+This component of the pVACtools workflow provides support for predicting
+neoantigens from gene fusions. Currently, fusion variants as reported by
+`INTEGRATE-Neo <https://github.com/ChrisMaherLab/INTEGRATE-Neo>`_ are supported, and predictions against any of the
+IEDB-supported prediction softwares can be reported.
```
Comment (chrisamiller) [inline docs/pvacfuse.rst]: s/predictions against any of the IEDB-supported prediction softwares can be reported./any of the IEDB-supported software can be used for binding prediction./

File: docs/pvacfuse/additional_commands.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
+.. image:: ../images/pVACfuse_logo_trans-bg_sm_v4b.png
+    :align: right
+    :alt: pVACfuse logo
+
+Additional Commands
+===================
+
+To make using pVACfuse easier several convenience methods are included in the package.
```
Comment (chrisamiller) [inline docs/pvacfuse/additional_commands.rst]: s/easier several/easier, several/

File: docs/pvacseq.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
+.. image:: images/pVACseq_logo_trans-bg_sm_v4b.png
+    :align: right
+    :alt: pVACseq logo
+
+.. _pvacseq:
+
+pVACseq
+=======
+
+pVACseq is a cancer immunotherapy pipeline for the identification of **p**\ ersonalized **V**\ ariant **A**\ ntigens by **C**\ ancer **Seq**\ uencing (pVACseq) that integrates tumor mutation and expression data (DNA- and RNA-Seq). It enables cancer immunotherapy research by using massively parallel sequence data to predicting tumor-specific mutant peptides (neoantigens) that can elicit anti-tumor T cell immunity. It is being used in studies of checkpoint therapy response and to identify targets for cancer vaccines and adoptive T cell therapies. For more general information, see the `manuscript published in Genome Medicine <http://www.genomemedicine.com/content/8/1/11>`_.
```
Comment (chrisamiller) [inline docs/pvacseq.rst]: "by cancer" or "in cancer"?

File: docs/pvacseq/additional_commands.rst [hunk 5]
```diff
@@ -1,7 +1,11 @@
+.. image:: ../images/pVACseq_logo_trans-bg_sm_v4b.png
+    :align: right
+    :alt: pVACseq logo
+
 Additional Commands
 ===================
 
-To make using pVAC-Seq easier several convenience methods are included in the package.
+To make using pVACseq easier several convenience methods are included in the package.
```
Comment (chrisamiller) [inline docs/pvacseq/additional_commands.rst]: s/easier several/easier, several/

**Comment by susannasiebert** [comment]
CI, test this.

**Pull request merged**